tests/unit/test-cmd/test.run.js

```diff
@@ -325,5 +326,101 @@ describe('run', () => {
            sourceDir: '/fake/source/dir',
          },
        );
-     });
-});
+     }
+  );
+
+  describe('profile-create-new option', () => {
+    beforeEach(() => {
+      sinon.stub(fs, 'mkdir');
+    });
+
+    afterEach(() => {
+      fs.mkdir.restore();
+    });
+
+    it('creates dir when firefox profile doesn\'t exist',
+       async () => {
+         sinon.stub(fs, 'existsSync').returns(false);
+         const firefoxProfile = '/pretend/path/to/Firefox/profile';
+         const cmd = prepareRun();
+
+         await cmd.run({
+           firefoxProfile,
+           profileCreateIfMissing: true,
+         });
+
+         sinon.assert.calledWith(fs.mkdir, firefoxProfile);
+
+         fs.existsSync.restore();
+       }
+    );
+
+    it('creates dir when chromium profile doesn\'t exist',
+       async () => {
+         sinon.stub(fs, 'existsSync').returns(false);
+         const chromiumProfile = '/pretend/path/to/Chromium/profile';
+         const cmd = prepareRun();
+
+         await cmd.run({
+           chromiumProfile,
+           target: 'chromium',
+           profileCreateIfMissing: true,
+         });
+
+         sinon.assert.calledWith(fs.mkdir, chromiumProfile);
+
+         fs.existsSync.restore();
+       }
+    );
+
+    it('Uses profile directory when firefox profile directory already exists',
```

Comment:
Nitpicks:
- `Uses` should be in lowercase, we may also tweak it into `uses the give firefox profile if it does exist`.
- we may want to assert that fs.mkdir was not called in this case
- stub and restore fs.existsSync in beforeEach and afterEach for the same reason described in my previous comment

Another thing that I've been thinking of is:
could these other two tests be refactored in a single parametrized test function as for the other two?